src/main/java/uk/gov/pay/api/model/telephone/State.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.pay.api.model.telephone;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class State {
+
+    @JsonProperty("status")
```

Comment:
you don't need these annotations if there is no difference between field name and json property name

Reply:
Done